From 280cf6d662e5c1521731d6248a887600c8cc41bb Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Wed, 13 May 2020 10:47:34 +0300
Subject: [PATCH] [SW-12964] goya2: pldm: skip btm fsm check

Seem like in pldm there is an issue with boot manager
fsm  which is not going back to idle state so ppboot get
stuck in while loop, waiting for it to finish.

same issue was also present in onix, pls refer
commit 612b6846de98 ("[SW-0] goya2: turn preboot_sync_btl weak")

Change-Id: If578fdad263e7c530c530f4575eeddd8d4a3b01a
Signed-off-by: Varun Sharma <vsharma@habana.ai>
---

diff --git a/goya2/pldm.c b/goya2/pldm.c
index 8a08f58..4408e08 100644
--- a/goya2/pldm.c
+++ b/goya2/pldm.c
@@ -38,5 +38,6 @@
 	return (factor + 1) & 0xFFFE;
 }
 
+void preboot_sync_btl(void) {}
 void preboot_init_pcie_phy(void) {}
 
